test/cluster/TypeDBClusterServerRunner.java

```diff
@@ -54,15 +54,14 @@ protected TypeDBClusterServerRunner createServerRunner(Map<String, String> optio
 
     class Standalone implements TypeDBClusterServerRunner {
 
-        private static final int ARCHIVE_INDEX = 1;
-
+        private static final String FLAG = "--server";
```

Comment:
Cluster is the server, we don't differentiate between cluster and core because we wouldn't run both at once as they share the same purpose.

Reply:
`FLAG` isn't a particularly meaningful name here. I would consider `ARCHIVE_PATH_PARAM`, but there's another issue just below...

This file really had quite a bit of black magic in it, didn't it? We need to dispel it where possible. On line 64 we do
```java
distribution = unarchive(getArchivePath(FLAG));
```
Even after renaming `FLAG`, it's still not at all clear what that line will do. We need to rethink this.

One possible band-aid would be renaming `FLAG` to `SERVER`. That makes line 64's intent clearer, but it dissolves the clarity of line 57. So the issue must run a little deeper.

As another perspective: given that there are only ever two kinds of archives, server and console, how about introducing methods named `getServerArchivePath` and `getConsoleArchivePath`?